src/App.test.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import store from './redux/store';
+import App from './App';
+
+test('renders learn react link', () => {
+  const { getByText } = render(
+    <Provider store={store}>
+      <App />
+    </Provider>,
+  );
+
+  expect(getByText(/learn/i)).toBeInTheDocument();
+});
```

Comment:
- [ ] Hi, please note that you should have unit & integration(using React Testing Library) tests for most of your components and some of your pure functions in the Redux code.